### DISCUSSION TITLE
Sequent SmartFan - Has anyone had to replace yet?

### DISCUSSION THREAD

**Discussion by johnny2678** [tagyoureit/nodejs-poolController#759] [General]
The Fan is still reporting fan speed but wondering if the fan is still turning?? 😂

Yea, it's FL, but we aren't in a sudden heat wave as the graph suggests.

![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/d66e10a9-102e-4264-a7d8-c2d988ae69c4)

I guess I could pull the stack but I'd have to disconnect everything I've wired up to it and I'm lazy so I thought i'd poll the group.  

Anyone had to replace yet?

**Comment by rstrouse** [comment]
You can replace the fan without replacing the board.

**Reply by johnny2678** [reply]
yep, I suppose I'll pull the stack and take a closer look.  Don't want a fried Pi 😂

**Comment by johnny2678** [comment]
Just went out for a closer look.  Based on CPU temps, The Sequent fan should be running at 100% right now.  It is spinning, but way slower than 100%.  I can see the individual blades turning.  Not even sure if these cheap little fans have ball bearings but probably some dust in the blades/gears/axle.  

From the sequent site, looks to be a 40x40x10mm 3 wire fan.  I'll look for replacements from the Bezos store and report back.

Luckily I had put in an 120mm enclosure fan on the exhaust port so that's providing at least some air flow.
![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/46152343-5d2b-46e6-87b9-490eae47768d)

**Reply by johnny2678** [reply]
I use Noctua fans in my PC builds - wonder if the this one could be swapped in? https://www.amazon.com/Noctua-Cooling-Blades-Bearing-NF-A4x10/dp/B009NQLT0M/ref=sr_1_10?crid=1FVNLX0U4X0YK&keywords=40x40x10+12v+fan+3pin&qid=1683903460&sprefix=40x40x10+12v+fan+3pin%2Caps%2C97&sr=8-10

**Comment by johnny2678** [comment]
follow up:
old stock fan on v2.0 sequent fan board was definitely burnt out

a quick retrofit with a 40x40x10 noctua fan and I'm back in business!  CPU temps have gone from 150 down to back below 120.

![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/bb521ed5-1d8b-4964-a3eb-f134ccccb236)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/5442612b-c26b-4fbd-9fac-cc76e0efdc38)

**Comment by johnny2678** [comment]
I'll leave this open so I can post follow ups in a couple of days for posterity.  hopefully the temps come way back down.

There was one slightly alarming issue that I noticed while doing this.  Will open a separate discussion.

**Comment by johnny2678** [comment]
follow up for anyone else that is running the Sequent Smartfan - the replacement Noctua fan works great.  Pi is running about 25 degrees above ambient in May, which is cooler than it was running in Feb!

![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/aa3f31db-5014-4581-985a-ec26efc8fd1e)
